modules/c++/six.sicd/include/six/sicd/Utilities.h

```diff
@@ -19,29 +19,140 @@
  * see <http://www.gnu.org/licenses/>.
  *
  */
-#ifndef __SIX_SICD_UTILITIES_H__
-#define __SIX_SICD_UTILITIES_H__
+#ifndef __SIX_SICD_UTILITIES_H__
+#define __SIX_SICD_UTILITIES_H__
 
 #include <memory>
 #include <string>
 #include <vector>
+#include <utility>
 
 #include <scene/SceneGeometry.h>
 #include <scene/ProjectionModel.h>
-#include <six/sicd/ComplexData.h>
-
-namespace six
-{
-namespace sicd
-{
-class Utilities
-{
-public:
+#include <six/sicd/ComplexData.h>
+
+#include <six/NITFReadControl.h>
+
+namespace six
+{
+namespace sicd
+{
+class Utilities
+{
+public:
     static scene::SceneGeometry* getSceneGeometry(const ComplexData* data);
-
-    static scene::ProjectionModel* getProjectionModel(const ComplexData* data, 
+
+    static scene::ProjectionModel* getProjectionModel(const ComplexData* data, 
             const scene::SceneGeometry* geom);
 
+   
+    /*
+     * getLoadedNitfReadControl creates and loads a NITFReadControl for the
+     * specified file.
+     *
+     * \param sicdPathname SICD NITF pathname
+     * \param schemaPaths One or more files or directories containing SICD
+     * schemas
+     *
+     * \return a NITFReadControl 'load()'ed with the given SICD
+     *
+     */
+    static
+    NITFReadControl getLoadedNITFReadControl(const std::string& sicdPathname,
```

Comment:
NITFReadControl is not safe to copy.  To that end, make it noncopyable (private, unimplemented copy constructor and assignment operator).  I have to look at the rest of the new code still, but if this function sticks around, have it return an auto_ptr.